packages/react-dom/src/client/ReactDOMFiberInput.js

```diff
@@ -190,6 +176,7 @@ export function updateWrapper(element: Element, props: Object) {
   }
 
   var value = props.value;
+  var valueAsString = '' + props.value;
```

Comment:
Nit: we might need it. So maybe we should only calculate it when necessary? Also this is a second `props.value` access but we've already just read it.

I would just leave inline conversions.

Reply:
Done in 5e194d1